code/modules/surgery/organs/dwarf_organ.dm

```diff
@@ -0,0 +1,46 @@
+/obj/item/organ/alcoholvessel //essentially the opposite of the xeno's plasmavessel, but with alcohol
```

Comment:
This organ is pretty redundant, the species is a better place to handle all this.